src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -151,7 +164,7 @@ const JobHistory = _.flow(
         })
       ]),
       div({ style: styles.submissionsTable }, [
-        !_.isEmpty(filteredSubmissions) && h(AutoSizer, [
+        !loading && hasJobs && h(AutoSizer, [
```

Comment:
In the case of refreshing the list, we'd like to show the old list in the background with the spinner on top, so we should remove the `!loading` clause here.